packages/store/src/workspace/page.ts

```diff
@@ -452,8 +452,12 @@ export class Page extends Space<FlatBlockMap> {
         yChildrenB.insert(nextIdx + 1, ids);
       }
     });
-    currentParent.propsUpdated.emit();
-    newParent.propsUpdated.emit();
+
+    currentParent.childrenUpdated.emit();
+    newParent.childrenUpdated.emit();
+    setTimeout(() => {
```

Comment:
The timeout here doesn't look right

Reply:
Yes, this is a temporary operation and has been fixed.